src/status_im/common/standard_authentication/standard_auth/slide_button/view.cljs

```diff
@@ -24,7 +24,7 @@
                                           :on-auth-success       on-auth-success
                                           :on-auth-fail          on-auth-fail
                                           :auth-button-label     auth-button-label}]))
-                         (into [] (concat [theme] dependencies)))]
+                         (vec (conj dependencies on-auth-success on-auth-fail)))]
```

Comment:
This functon is wrapped on `rn/use-callback` but it wasn't receiving the `on-auth-success` callback as a dependency, leading to potential bugs (I faced one), now I added it